lib/index.js

```diff
@@ -449,3 +436,14 @@ module.exports.element = new Proxy(element, {
     return global.browser.element[name];
   }
 });
+
```

Comment:
why is this change required?

Reply:
`tsc` was throwing some error earlier related to overwriting `browser` and `app` after defining them using `Object.defineProperty('browser'/'app')` (if I can remember correctly). But I'm unable to reproduce that error now.

Reply:
In that case, revert this for now and we can look at if again if we face any issues. I don't like putting anything after the export statements.

Reply:
Okay

Reply:
Done.

Reply:
I'm able to reproduce the error locally as well by running `npm run test`. It seems the problem is with the use of `"use strict";` at the top of the compiled JS file which is not allowing to set the `browser` property again using `module.exports.browser = browser;`.

Reply:
Should I revert the last commit, cause I don't think there's any other good way to get past this? Also, I think it makes sense because `Nightwatch.browser` is being overwritten using `Object.defineProperty()`, so it should come below the earlier definition.

Reply:
Yeah revert it.